### PR TITLE
add MiModD suite

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
depending on https://github.com/usegalaxy-eu/galaxy-playbook-temporary/pull/193

File: tools.yaml [hunk 1]
```diff
@@ -4,17 +4,21 @@ install_resolver_dependencies: true
 install_tool_dependencies: false
 
 tools:
+  - name: suite_mimodd
```
Comment (wm75) [inline tools.yaml]: mimodd_main

would be better than suite_mimodd (the suite really brings in only stuff that's unnecessary on a well-equipped server; you don't need another aligner, nor another SnpEff wrapper I would say)

**Comment by erasche** [comment]
(merging now, it won't get installed until this weekend)

File: tools.yaml (hunk 1 above)
Comment (bgruening) [reply tools.yaml]: Mh, that is confusing ;)

Comment (bgruening) [reply tools.yaml]: Changed it on master.

**Comment by wm75** [comment]
After things got installed you may want to have a look at
http://mimodd.readthedocs.io/en/latest/install_toolshed.html#enhancing-the-user-experience .
The display whitelisting would be most important.

In addition, it would be great if you could configure the job runner for the Variant Calling tool to enable
multiprocessing. The tool parallelizes variant calling by contig and since most of our end users are working with C. elegans, which has 6 chromosomes, 6 cores would be an ideal value.

File: tools.yaml (hunk 1 above)
Comment (wm75) [reply tools.yaml]: perfect, thanks.

**Pull request merged**